.changeset/eight-pillows-search.md

```diff
@@ -0,0 +1,5 @@
+---
+'@talend/design-system': major
```

Comment:
🤣

Reply:
See PR description 🤷

Reply:
Try a minor first. Yes it's technically a breaking but it's never been used.

Reply:
<img width="1508" alt="image" src="https://user-images.githubusercontent.com/18534166/165525117-96ce6f26-7778-43f2-8c47-f238e5b4aafe.png">

Not used for now

Reply:
```suggestion
'@talend/design-system': minor
```